homeassistant/components/recorder/statistics.py

```diff
@@ -11,6 +11,7 @@
 import logging
 from operator import itemgetter
 import re
+from time import time as time_time
```

Comment:
Imported as `time_time` because
```
homeassistant/components/recorder/statistics.py:992:26: W0621: Redefining name 'time' from outer scope (line 14) (redefined-outer-name)
homeassistant/components/recorder/statistics.py:1041:27: W0621: Redefining name 'time' from outer scope (line 14) (redefined-outer-name)
homeassistant/components/recorder/statistics.py:1099:28: W0621: Redefining name 'time' from outer scope (line 14) (redefined-outer-name)
```